CNAME

```diff
@@ -0,0 +1 @@
+rnawealth.andydiary.com
```

Comment:
require fixed

Reply:
We need create CNAME file for github page.